crates/bootstrap_srv/src/bin/kitsune2-bootstrap-srv.rs

```diff
@@ -5,19 +5,27 @@ use kitsune2_bootstrap_srv::*;
 #[derive(clap::Parser, Debug)]
 #[command(version)]
 pub struct Args {
-    /// By default kitsune2-boot-srv runs in "testing" configuration
+    /// By default, kitsune2-boot-srv runs in "testing" configuration
     /// with much lighter resource usage settings. This testing mode
     /// should be more than enough for most developer application testing
     /// and continuous integration or automated tests.
     ///
-    /// To setup the server to be ready to use most of the resources available
+    /// To set up the server to be ready to use most of the resources available
     /// on a single given machine, you can set this "production" mode.
     #[arg(long)]
     pub production: bool,
+
     /// Output tracing in json format.
     #[arg(long)]
     pub json: bool,
-    // TODO - Implement the ability to specify TLS certificates
+
+    /// The path to a TLS certificate file.
+    #[arg(long, requires = "tls_key")]
+    pub tls_cert: Option<std::path::PathBuf>,
+
+    /// The path to a TLS key file.
+    #[arg(long, requires = "tls_cert")]
+    pub tls_key: Option<std::path::PathBuf>,
```

Comment:
`tls_cert_pem_file` + `tls_key_pem_file` just to be [explicit](https://github.com/holochain/sbd/blob/main/rust/sbd-server/src/config.rs#L14-L26)?

Reply:
Okay to solve with documentation on the CLI flag rather than a longer flag? Otherwise happy to make the rename and be consistent

Reply:
Yep, I'm cool with documentation :+1: